frontend/pages/hosts/HostDetailsPage/PackQueriesListRow/PackQueriesListRow.jsx

```diff
@@ -19,11 +19,24 @@ class PackQueriesListRow extends Component {
       last_executed,
     } = query;
 
+    const secondsToHms = (d) => {
```

Comment:
This function could live in the [kolide/helpers.ts](https://github.com/fleetdm/fleet/blob/master/frontend/kolide/helpers.ts). That way, if we end up needing to do a similar seconds-to-hours/minutes/seconds conversion in a different file, we can import your function.

Reply:
Ok, I'll get on that today. Ty!